module Main in examples;

@Start
defun main (args : String[]) : void
{
    # This throw will crash the entire program,
    # because the exception is caught by the runtime.

    throw new Exception();
}